### PR TITLE
Document node rescue mode

### PR DESCRIPTION
**Pull request by tzumainn**

### CODE REVIEW

File: docs/source/usage/cli.rst [hunk 1]
```diff
@@ -308,6 +308,24 @@ If ``console_enabled`` is ``true``, we can access the serial console using follo
 If ``console_enabled`` is ``false`` or ``console_info`` is ``None`` then
 the serial console is disabled. Note, there can only be one ipmi connection to the node, meaning only one user may access the console at a time.
 
+Rescue Mode
+-----------
+
+A node can be booted in rescue mode, potentially allowing for some form of remediation if some aspect of the node fails. Doing so puts the node
+on the rescue network temporarily to allow the rescue image to boot; then the node is returned to its original network configuration. If the node
+has an external floating IP attached, you can then access the node by running ``ssh rescue@<floating-ip>`` and using the password specified in
+the rescue command.
```
Comment (naved001) [inline docs/source/usage/cli.rst:317]: can we make a note that the password specified should be a strong password?

**Review by naved001 — APPROVED** [review]
looks good to me, just one comment inline.

File: docs/source/usage/cli.rst (hunk 1 above)
Comment (tzumainn) [reply docs/source/usage/cli.rst]: Seems very reasonable - done!

**Review by naved001 — APPROVED** [review]

**Review by DanNiESh — APPROVED** [review]
Looks good to me!

**Pull request merged**